src/select-menu/src/SelectMenu.js

```diff
@@ -192,23 +78,139 @@ export default class SelectMenu extends PureComponent {
             isMultiSelect={isMultiSelect}
             titleView={titleView}
             listProps={{
-              onSelect: item => {
-                this.props.onSelect(item)
-              },
-              onDeselect: item => {
-                this.props.onDeselect(item)
-              },
-              onFilterChange: this.props.onFilterChange,
+              onSelect,
+              onDeselect,
+              onFilterChange,
               selected: arrify(selected)
             }}
             close={close}
-            {...this.getDetailView(close, detailView)}
-            {...this.getEmptyView(close, emptyView)}
+            {...getDetailView(close, detailView)}
+            {...getEmptyView(close, emptyView)}
             closeOnSelect={closeOnSelect}
           />
         )}
-        {...props}
+        {...rest}
+        ref={ref}
```

Comment:
`<Popover />` will be a really tricky one to convert, since so much stuff is reliant upon it. Right now, it actually swallows refs, so this is a no-op. This just preserves the same prop behavior as before.

Reply:
Do we want to avoid ref forwarding in this component then?

Reply:
Probably. I wanted to decouple API changes from the conversion piece just to make it a little easier to read

Reply:
But definitely worth a follow-up. Might make an `API Changes` section in the v5 tracker